rqt_reconfigure/src/rqt_reconfigure/treenode_qstditem.py

```diff
@@ -103,6 +103,11 @@ def __init__(self, *args):
         except IndexError:  # tuple index out of range etc.
                 rospy.logerr('TreenodeQstdItem IndexError')
 
+    def __del__(self):
```

Comment:
Since Python won't call this in a deterministic manner this should be done in an explicitly called method. How does that relate to the method `clear_dynreconf_client`?